app/less/shipyard.less

```diff
@@ -20,6 +20,7 @@ a.ship {
     width: 100%;
     margin: 0;
     text-align: left;
+    font-size: 1.1em;
```

Comment:
That's fine for now. Going to change some of the shipyard page layout later tonight